extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -136,6 +150,12 @@ export class AmpStoryPage extends AMP.BaseElement {
     /** @private {?Element} */
     this.playMessageEl_ = null;
 
+    /** @private {?Element} */
+    this.expandedViewOverlay_ = null;
```

Comment:
Can this be moved to the story level?  Then dynamically appended to the currently active page?  That way, there will only be one per story.